### DISCUSSION TITLE
浏览器插件能登录，ios端的app，同样api，无法登录，提示上面问题。

### DISCUSSION THREAD

**Discussion by star2k** [usememos/memos#1623] [Q&A]
![image](https://user-images.githubusercontent.com/9582078/235645328-0da681b2-eadf-4c18-bf3a-2af1a5df2f4a.png)

**Comment by Zeng1998** [comment]
need https.

**Comment by adll123** [comment]
官网的FAQ有写：
The App Transport Security policy of iOS requires HTTPS for your security. It’s recommended to use [acme.sh](https://github.com/acmesh-official/acme.sh) to enable HTTPS on your server.
